std/traits.d

```diff
@@ -7097,19 +7097,49 @@ if (T.length == 1)
 }
 
 /**
-Determines whether function $(D f) requires a context pointer.
+Determines whether function `f` requires a context pointer.
+
+Params:
+    f = a function
+Returns
+    A `bool`
 */
 template isNestedFunction(alias f)
 {
-    enum isNestedFunction = __traits(isNested, f);
+    enum isNestedFunction = __traits(isNested, f) && isSomeFunction!(f);
 }
 
+///
 @safe unittest
 {
-    static void f() { }
-    void g() { }
```

Comment:
changed up this example because it's not clear to the documentation user that the example happens in a function context